pipeline/terraform/stack/elastic_pipeline.tf

```diff
@@ -180,8 +180,8 @@ locals {
     "relation_embedder",
     "router",
     "inferrer",
-    "ingestor_works",
-    "ingestor_images",
+    "work_ingestor",
```

Comment:
Why'd we change this? Would it make more sense to pluralise it?

Reply:
because it was not in sync with what is created in the create_elasticsearch_users script. We could pluralize, but we have to do it there too. I just opted for the simplest change which was this